package.json

```diff
@@ -96,6 +97,98 @@
             "type": "string"
           },
           "markdownDescription": "Optional list of custom resolvers passed to Coursier when fetching metals dependencies.\n\nFor documentation on accepted values see the [Coursier documentation](https://get-coursier.io/docs/other-repositories).\n\nThe extension will pass these to Coursier using the COURSIER_REPOSITORIES environment variable after joining the custom repositories with a pipe character (|)."
+        },
+        "treeviews.initialWidth": {
+          "type": "number",
+          "default": 40,
+          "description": "Initial TreeViews panel width"
+        },
+        "treeviews.alignment": {
+          "type": "string",
+          "default": "left",
+          "enum": [
+            "left",
+            "right"
+          ],
+          "description": "TreeViews panel position"
+        },
+        "treeviews.initialViews": {
+          "type": "object",
+          "default": [
+            {
+              "name": "metalsCompile",
+              "size": 10,
+              "expanded": [
+                [
+                  "metals://ongoing-compilations"
+                ]
+              ]
+            },
+            {
+              "name": "metalsBuild",
+              "size": 40,
+              "expanded": [
+                [
+                  "projects:"
+                ]
+              ]
+            }
+          ]
+        },
+        "treeviews.toggleNode": {
+          "type": "string",
+          "default": "<CR>",
+          "description": "Expand/Collapse tree node"
+        },
+        "treeviews.forceChildrenReload": {
+          "type": "string",
+          "default": "f",
+          "description": "Force reloading children of this node. Maybe be useful when wrong result is cached and tree contains invalid data."
+        },
+        "treeviews.gotoParentNode": {
+          "type": "string",
+          "default": "p",
+          "description": "Go to parent node"
+        },
+        "treeviews.gotoFirstChild": {
+          "type": "string",
+          "default": "K",
```

Comment:
Ahh, that makes sense. So actually playing around with it a bit now, going to `G` would be a terrible ideas since it'd hijack my ability to jump to the bottom. For the keys, I'd say I'll need to use this a bit for a while before figuring out what makes most sense. 👍